site/docs/blog/namespace-enforcement.md

```diff
@@ -0,0 +1,7 @@
+# Namespace enforcement in Nessie
+
+Starting from Nessie version 0.52.3, it is required to have existing Namespaces before creating or committing to tables.
+<br>In case of tables that have implicit namespaces, new commits will fail after the upgrade as this rule will be enforced.
```

Comment:
Why not use normal markdown format (not HTML)?

Reply:
using the markdown syntax now.